cookbooks/bcpc/recipes/haproxy.rb

```diff
@@ -67,7 +67,8 @@
 template "/etc/haproxy/haproxy.cfg" do
   source "haproxy.cfg.erb"
   mode 00644
-  variables(:mysql_servers => get_nodes_for("mysql","bcpc"))
+  variables(:mysql_servers => get_nodes_for("mysql","bcpc"),
+    :oozie_servers => get_nodes_for("oozie", "bcpc-hadoop"))
```

Comment:
This is something I can look into. It would definitely be nice to break the configuration for each each new service.